deploy/helm/magda/charts/session-db/templates/statefulset.yaml

```diff
@@ -14,6 +14,7 @@ spec:
       labels:
         service: session-db
     spec:
+      priorityClassName: magda-10
```

Comment:
I don't think the session db needs to be higher than the other dbs - if it goes down most things will (should?) still work, it's just signed in functionality that'll fail from a user's point of view, and stuff within the system like crawling, which can wait.